### PR TITLE
Doubly linked list

### PR DESCRIPTION
**Pull request by lumodon**

### CODE REVIEW

File: src/doublyLinkedList.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  this._next = null
+  this._prev = null
+
+  this.data = () => this._data
+  this.next = () => this._next
+  this.prev = () => this._prev
+}
+
+export default class LinkedList {
+
```
Comment (devonwesley) [inline src/doublyLinkedList.js:14]: Should this be named DoublyLinkeList??

File: src/doublyLinkedList.js [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+function Node(value){
+  this._data = value
+  this._next = null
+  this._prev = null
+
+  this.data = () => this._data
+  this.next = () => this._next
+  this.prev = () => this._prev
+}
+
```
Comment (devonwesley) [inline src/doublyLinkedList.js:12]: What are the purpose of these functions? shouldn't we just have the underscored properties and set the value directly to those?

File: src/doublyLinkedList.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  remove(){
+    let currentNode = this._head
+
+    if(!currentNode){
+      return null
+    }
+
+    while(currentNode.next().next() !== null){
+      currentNode = currentNode.next()
```
Comment (devonwesley) [inline src/doublyLinkedList.js:177]: Does this function not work with one .next()?

File: src/doublyLinkedList.js (hunk 1 above)
Comment (shakalee14) [reply src/doublyLinkedList.js:14]: yes lol

File: src/linkedList.js [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+  this.next = () => this._next
+}
+
+export default class LinkedList {
+
+  constructor() {
+    this._length = 0
+    this._head = null
+    this._tail = null
+  }
```
Comment (devonwesley) [inline src/linkedList.js:17]: A normal linked list does not have a tail.

File: src/doublyLinkedList.js (hunk 2 above)
Comment (shakalee14) [reply src/doublyLinkedList.js:12]: use them as pointers.

File: src/linkedList.js (hunk 4 above)
Comment (shakalee14) [reply src/linkedList.js:17]: thanks!

**Pull request merged**